ui/packages/catalog-ui-search/src/main/webapp/component/ingest-editor/ingest-editor.less

```diff
@@ -0,0 +1,42 @@
+@{customElementNamespace}ingest-editor {
+  .customElement;
+  animation: new-ingest-editor .250s 1 ease-in-out;
+  padding: 10px;
```

Comment:
❓ Doesn't pixel padding have scaling issues? I thought we preferred `em` units.

Reply:
Changed to use em